met-web/src/components/engagement/dashboard/comment/CommentViewContext.tsx

```diff
@@ -36,6 +54,11 @@ export const CommentViewProvider = ({ children }: { children: JSX.Element | JSX.
     const [isEngagementLoading, setEngagementLoading] = useState(true);
     const [isCommentsListLoading, setIsCommentsListLoading] = useState(true);
     const [comments, setComments] = useState<Comment[]>([]);
+    const [paginationOptions, setPaginationOptions] = useState<PaginationOptions<Comment>>(initialPaginationOptions);
+    const [pageInfo, setPageInfo] = useState<PageInfo>(initialTableLoading);
```

Comment:
I think you meant initialPageInfo, also, I think i saw default/initial values being set a few times, once on each component, i'm just throwing it out there there it might be better to have it centralized somewhere, with typescript classes this is easily handled by constructors, but since we are using interfaces I had created a method to retrieve default values for the engagement model, if you have a better idea let me know!

Reply:
Thanks good eye, I added a method to create default page info, but for PaginationOptions it's a bit more difficult since the default fields are not the same for different pages and also it is a typed interface, it's easier to keep it flexible depending on the page I thought.